### DISCUSSION TITLE
v0.6.4

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#369] [General]
Small release with a bunch of important bugfixes.

Changes:
- Fixed comment width displaying incorrectly
- Fixed [display] object
- Fixed flipped y coordinates on [pad], [2dslider], [multi.vsl], [canvas.mouse] and more
- Fixed [function] losing send/receive symbols
- Fixed document browser repaint bug
- Fixed Heavy console output getting cut off
- Fixed send/receive bug for float/symbol/list box
- Fixed shift+drag crash
- Fixed bug causing patch to become unscrollable after zooming
- Fixed [pad] responding to mouse events when it's outside of graph bounds
- Fixed graphical glitch in inspector
- [pic] now responds to "open" message
- Added missing cyclone objects
- Fixed bug with C++ exporter
- Added some cyclone docs (A to C alphabetically)
- Updated pd-lua to 0.11.2
- Linux: Fixed bug causing shadow to be drawn on top of suggestions component
- Linux: Fixed popupmenu background when there's no transparency allowed
- MacOS: Installer no longer asks to install Rosetta 2
- Windows: No longer depends on the Visual C++ Redistributable (but issue #355 is still not resolved)
- Simplified build system

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/v0.6.4'>v0.6.4</a>.</em>

**Comment by jcsekinger** [comment]
unable to install it in Manjaro.
![Capture d’écran du 2023-01-30 08-04-11](https://user-images.githubusercontent.com/12682457/215688173-47a14260-7f28-4258-805b-d01e2991ec60.png)

**Reply by timothyschoen** [reply]
Whoops, let me fix that!

**Reply by jcsekinger** [reply]
It’s working now ! thank you very much !
